### ISSUE TITLE
 Permission denied install zip

### ISSUE THREAD

**Issue by qiurigao** [TeamWin/Team-Win-Recovery-Project#956]
I:Update binary zip
I:Zip does not contain SELinux file_contexts file in its root.
I:Legacy property environment initialized.
Target: vivo y55 16.6 no verify boot
verifying secure boot image...
package_extract_file: can't open /dev/block/bootdevice/by-name/boot for write: Permission denied
package_extract_file: can't open /dev/block/bootdevice/by-name/recovery for write: Permission denied
Patching TWRP image unconditionally...
overwatch Bootloader
script succeeded: result was [overwatch Bootloader]I:Updater process ended with RC=0
I:Legacy property environment disabled.
I:Install took 0 second(s).

**Comment by qiurigao** [comment]
My Build TWRP is good ,but not flash zip to emmc img

**Comment by junzhengca** [comment]
Seems like this is a problem with the .zip, maybe post the issue on their repo?

**Comment by qiurigao** [comment]
NO!  I install Magisk-v13.3 and SuperSU Is also a similar error, no permission to write boot.

**Comment by imharsher** [comment]
Hey can you  help me how to root vivo y55s device

**Comment by shaggyskunk** [comment]
Wrong place to ask - Search xda.

**Comment by imharsher** [comment]
I also searched on XDA

**Comment by imharsher** [comment]
But i got nothing

**Comment by shaggyskunk** [comment]
regardless - This is NOT the place to get help on Rooting.
